neurom/features/morphology.py

```diff
@@ -608,3 +604,83 @@ def get_points(neurite):
     total_volume = sum(iter_neurites(morph, mapfun=nf.total_volume, filt=is_type(neurite_type)))
 
     return total_volume / morph_hull.volume
+
+
+def _unique_projected_points(morph, projection_plane,  neurite_type):
+
+    key = "".join(sorted(projection_plane.lower()))
+
+    try:
+        axes = {"xy": COLS.XY, "xz": COLS.XZ, "yz": COLS.YZ}[key]
+
+    except KeyError as e:
+
+        raise NeuroMError(
+            f"Invalid 'projection_plane' argument {projection_plane}. "
+            f"Please select 'xy', 'xz', or 'yz'."
+        ) from e
+
+    points = list(
+        iter_neurites(morph, mapfun=sf.section_points, filt=is_type(neurite_type))
+    )
+
+    if len(points) == 0:
+        return np.empty(shape=(0, 3), dtype=np.float32)
+
+    return np.unique(np.vstack(points), axis=0)[:, axes]
+
+
+@feature(shape=())
+def aspect_ratio(morph, neurite_type=NeuriteType.all, projection_plane="xy"):
+    """Calculates the min/max ratio of the principal direction extents along the plane.
+
+    Args:
+        morph: Morphology object.
+        neurite_type: The neurite type to use. By default all neurite types are used.
+        projection_plane: Projection plane to use for the calculation. One of ('xy', 'xz', 'yz').
+
+    Returns:
+        The aspect ratio feature of the morphology points.
+    """
+    projected_points = _unique_projected_points(morph, projection_plane, neurite_type)
+    return [] if len(projected_points) == 0 else morphmath.aspect_ratio(projected_points)
+
+
+@feature(shape=())
+def circularity(morph, neurite_type=NeuriteType.all, projection_plane="xy"):
+    """Calculates the circularity of the morphology points along the plane.
+
+    The circularity is defined as the 4 * pi * area of the convex hull area over its
```

Comment:
```suggestion
    The circularity is defined as the 4 * pi * area of the convex hull over its
```
Shouldn't this word be removed?

Reply:
yeah